WORKSPACE

```diff
@@ -103,6 +103,21 @@ load("@publish_deps//:requirements.bzl", "install_deps")
 
 install_deps()
 
+#####################
+# Install sphinx for doc generation.
+load("@python//3.11.6:defs.bzl", "interpreter")
+load("@rules_python//python:pip.bzl", "pip_parse")
+
+pip_parse(
```

Comment:
nit: we could add the equivalent to `MODULE.bazel` and mark the `sphinx` deps as `dev_dependencies` and support this under `bzlmod`.

However, I am not sure if we have support for `module_ctx.is_dev_dependency` in the `pip.parse`. If not, then it is fine to leave as is.

Reply:
For some weird reason, bzlmod isn't working with this.

```
ERROR: Analysis of target '//docs/sphinx:docs' failed; build aborted: module extension "toolchains" from "@rules_java~6.3.0//java:extensions.bzl" does not generate repository "remote_java_tools_darwin", yet it is imported as "remote_java_tools_darwin" in the usage at bazel_tools@_/MODULE.bazel:20:32
```

Which is weird, because didn't we just enable bzlmod + stardoc a few weeks ago? I'll have to poke this a bit more.

Reply:
I think the reason is that `.bazelversion` is `6.0.0` which is super old. Changing it to `6.3.2` allows me to run `:docs.serve`.

Reply:
Thanks. Yes, that was it. I updated it to 6.2.0, since that's the lowest version that things seem happy with.